libs/blocks/media/media.js

```diff
@@ -33,6 +33,14 @@ function decorateQr(el) {
   qrImage.classList.add('qr-code-img');
 }
 
+export async function loadCountDownTimer(el) {
```

Comment:
I see the same method in both marquee and media; what do you think about adding it to `decorate.js` and load it conditionally?